src/config/index.js

```diff
@@ -1,2 +1,43 @@
 export const DEFAULT_CURRENCY = "USD";
+ 
+//Date configs -- START
+export const DATE_FORMATS = [
+  "dd MMMM, yyyy",
+  "dd/MM/yyyy",
+  "dd.MM.yyyy",
+  "dd-MM-yyyy",
+  "dd/MM/yy",
+  "dd.MM.yy",
+  "dd-MM-yy"
+];
 
+export const TIME_FORMATS = ["HH:mm", "HHmm", "HH-mm", "HH mm"];
+
+export const DATE_TIME_FORMATS = [
```

Comment:
As we have two different controls for date and time, do we really need this? This is basically all permutations of DATE and TIME formats.

Reply:
removed this